src/command_modules/azure-cli-datalake/azure/cli/command_modules/datalake/store/custom.py

```diff
@@ -0,0 +1,275 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from azure.mgmt.datalake.store.models import (DataLakeStoreAccountUpdateParameters,
+                                              FirewallRule,
+                                              DataLakeStoreAccount,
+                                              EncryptionConfigType,
+                                              EncryptionIdentity,
+                                              EncryptionConfig,
+                                              EncryptionState,
+                                              KeyVaultMetaInfo)
+
+from azure.datalake.store.multithread import (ADLUploader, ADLDownloader)
+from azure.cli.command_modules.datalake.store._client_factory import (cf_datalake_store_filesystem)
+from azure.cli.core.commands.client_factory import get_mgmt_service_client
+from azure.cli.core._util import CLIError
+import azure.cli.core.azlogging as azlogging
+
+logger = azlogging.get_az_logger(__name__)
+# account customiaztions
+def list_adls_account(client, resource_group_name=None):
+    account_list = client.list_by_resource_group(resource_group_name=resource_group_name) \
+        if resource_group_name else client.list()
+    return list(account_list)
+
+# pylint: disable=too-many-arguments
+def create_adls_account(client,
+                        resource_group_name,
+                        account_name,
+                        location=None,
+                        default_group=None,
+                        tags=None,
+                        encryption_type=EncryptionConfigType.service_managed,
```

Comment:
When you use enums as defaults, it should be "{enum_type}.{enum_value}.value".  Otherwise when you do -h you'll see "Default: EncryptionConfigType.ServiceManaged"  which won't even match the values in Allowed.